### DISCUSSION TITLE
Why isn't history file a plain text file?

### DISCUSSION THREAD

**Discussion by Aster89** [qutebrowser/qutebrowser#7867] [Q&A]
Yeah, just that. I'm curious.

I bet the answer is performance. But why? What's so problematic in the history file being a plain text file with one entry per line? Each entry would be url and date-time.

**Comment by The-Compiler** [comment]
It used to be one until v1.0.0. Indeed the problem was performance - both reading the file on startup as well as actually filtering it.
